cmd/thanos/receive.go

```diff
@@ -173,6 +175,15 @@ func runReceive(
 		return errors.Wrapf(err, "migrate legacy storage in %v to default tenant %v", conf.dataDir, conf.defaultTenantID)
 	}
 
+	relabelContentYaml, err := conf.relabelConfigPath.Content()
+	if err != nil {
+		return errors.Wrap(err, "get content of relabel configuration")
+	}
+	var relabelConfig []*relabel.Config
+	if err := yaml.Unmarshal(relabelContentYaml, &relabelConfig); err != nil {
+		return errors.Wrap(err, "parsing relabel configuration")
```

Comment:
Tiiiiny nit: I think that by convention we use the present tense rather than the gerund when wrapping errors:
```suggestion
		return errors.Wrap(err, "parse relabel configuration")
```